components/Curated.react.tsx

```diff
@@ -1,16 +1,20 @@
 import React, { useEffect, useRef, useState } from "react";
 import { Container, ResponsiveEmbed } from "react-bootstrap";
-import { StyleSheet, css } from "aphrodite";
 import { useMediaQuery } from "react-responsive";
 import SaferLink from "./SaferLink.react";
+import styles from "../styles/modules/Curated.react.module.css";
 
 const VideoThumbnail = ({ videoID, title }) => {
   const [isModalOpen, setIsModalOpen] = useState(false);
-  const dialogRef = useRef<HTMLDialogElement | null>(null);
-  const dialogIFrameRef = useRef<HTMLIFrameElement | null>(null);
+  const dialogRef = useRef(null);
```

Comment:
I don't see how removing these type annotations is a good thing.

Reply:
lol